armi/reactor/components/volumetricShapes.py

```diff
@@ -250,7 +254,11 @@ def getComponentVolume(self):
         return vol
 
     def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
-        return self.getDimension("outer_radius", Tc, cold)
+        return 2.0 * self.getDimension("outer_radius", Tc, cold)
```

Comment:
Oops... is this method only used for sorting?

Reply:
I think only for sorting, yes